docs/serializable.md

```diff
@@ -199,4 +206,33 @@ assertEquals(
 );
 ```
 
-[Strategies](./strategies) are ways to convert data to and from a JSON value.
\ No newline at end of file
+## Short cutting the `@SerializeProperty` decorator
+
+While `@SerializeProperty` is handy with to and from JSON strategies, it can
+still be verbose to declare the strategies for each property.
+`@SerializeProperty` can have shortcut functions that provide the to and from
+JSON strategies accepting arguments. An example short cut is providing a `type`
+to use with `toSerializable`:
+
+```ts
+export function DeserializeAs<T>(
+  type: T & { new (): Serializable },
```

Comment:
yea, now I question the name of it.... arg _naming things_... do you have a suggestion? as it can be used with decorators as well... `SerializableType<T>`??

Reply:
SerializableConstructor<T>?

Reply:
noicce, changed

Reply:
changed `getNew` to `getNewSerializable` which I think is more clear about what it does